audbackend/core/artifactory.py

```diff
@@ -26,6 +24,11 @@ def __init__(
     ):
         super().__init__(host, repository)
 
+        self.root = f'{self.host}/{self.repository}'
+        path = audfactory.path(self.root)
+        if not path.exists():
+            audfactory.path(self.root).mkdir()
```

Comment:
This does not create a repository, this creates a folder in an existing repository.

Reply:
I think code to create repositories looks more like the examples at https://github.com/devopshq/artifactory#repositorylocal, even though I don't understand them yet.

Reply:
Ah great, I was already wondering how to create a "real" repository.

What is a bit problematic at the moment is that in our tests we actually use a folder on a repo as repo. But I guess that is anyway bad practice and we should change it once we are able to properly create a repository inside the backend. Let's see...

Reply:
I guess we should then forbid to use a folder in a repo as repo, right?

Reply:
In general it doesn't seem to be a problem as it works. But it is not a repository, as a repository has special features on Artifactory (e.g. it can be mirrored and stuff like this). So yes, we should forbid to use a folder as repository on the Artifactory backend.

Reply:
I have now changed the code to create a repository instead of a folder inside an existing one. Actually it is no longer possible to initialize an Artifactory backend with a folder and a (failing) test has been added.

Reply:
I cannot find any leftover repositories on Artifactory, so it seems to work:

![image](https://user-images.githubusercontent.com/173624/233064350-ab824c5c-93f7-4dd5-acd1-4b45f40c1983.png)